kubernetes/start/suse/adoc/gs_rancher_edge-analytics_finance_stocks.adoc

```diff
@@ -1,21 +1,85 @@
 :docinfo:
 include::./common_docinfo_vars.adoc[]
 
-:title: Edge Analytics with SUSE Rancher: Finance - Market Predictions : Getting Started
-:productname: SUSE Rancher 2.5
-:productversion: 2.5
-:productname2: K3s v1.20.14+k3s1
-:productversion2: v1.20.14+k3s1
-:partnerproductname: Fin-Dashboard
-:pythonname: Python 3.10
-:author1: Navin Chandra, Google Summer of Code Contributor, openSUSE Project
+// = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = =
+// General comments
+// Keep in mind that this is a "getting started" guide and the
+//   audience that you are trying to reach.
+// Leverage ASCIIDoc features to make this document readable and usable:
+//   - Text highlights (follow SUSE style guides)
+//   - Admonitions (i.e., NOTE, TIP, IMPORTANT, CAUTION, WARNING)
+//   - Code blocks
+//   - Lists (ordered and unordered, as appropriate)
+//   - Links (to other resources)
+//   - Images
+//     - Place image files under the ./media directory tree
+//       (e.g., ./media/src/svg, ./media/src/png)
+//     - Format preference: svg > png > jpg
+//     - Consolidate images wherever possible
+//       (i.e., don't use two images when one conveys the message)
+//   - Use sections and subsections to organize and group related
+//     steps.
+//
+// = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = =
+
+
+
+// = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = =
+// Variables & Attributes
+//
+// NOTES:
+// 1. Update variables below and adjust docbook file accordingly.
+// 2. Comment out any variables/attributes not used.
+// 3. Follow the pattern to include additional variables.
+//
+// = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = =
+
+// - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+// organization - do NOT modify
+:trd: Technical Reference Documentation
+:type: Getting Started
+// - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+
+
+// - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+// document and product
+:title: Edge Analytics in Finance
+:subtitle: Market Predictions
+:productname: Rancher 2.6
+:platform1: Rancher 2.6 by SUSE
+:platform2: SUSE Linux Enterprise Server 15 SP3
+:otherproduct1: Fin-Dashboard
+:usecase: edge analytics in finance
+// - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+
+
+// - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+// authors
+:author1_firstname: Navin
+:author1_surname: Chandra
+:author1_jobtitle: Google Summer of Code 2022 Contributor
+:author1_orgname: openSUSE
+// :author1_email: abhinavsharma332@gmail.com
```

Comment:
Correct Email to - navinchandra772@gmail.com

Reply:
Thanks for catching this, Navin.